numpy/lib/function_base.py

```diff
@@ -1711,12 +1711,17 @@ def diff(a, n=1, axis=-1):
         The number of times values are differenced.
     axis : int, optional
         The axis along which the difference is taken, default is the last axis.
+    to_begin : 1d array_like
+        Values to insert along axis before the differenced results
+    to_end : 1d array_like
+        Values to insert along axis after the differenced results
 
     Returns
     -------
     diff : ndarray
         The n-th differences. The shape of the output is the same as `a`
-        except along `axis` where the dimension is smaller by `n`.
+        except along `axis` where the dimension is smaller by `n`, and possibly 
+        increased by len(to_begin) + len(to_end)
```

Comment:
Rather just use words "increased by the lengths of ``to_begin`` and ``to_end``" since it isn't necessary the `len()` by which the size will increase.